src/dhcpmon/src/dhcp_device.c

```diff
@@ -353,29 +353,38 @@ static dhcp_mon_status_t dhcp_device_check_health(dhcp_mon_check_t check_type,
 }
 
 /**
- * @code dhcp_print_counters(vlan_intf, counters);
+ * @code dhcp_print_counters(vlan_intf, type, counters);
  *
  * @brief prints DHCP counters to sylsog.
  *
+ * @param vlan_intf vlan interface name
+ * @param type      counter type
  * @param counters  interface counter
+ *
+ * @return none
  */
-static void dhcp_print_counters(const char *vlan_intf, uint64_t counters[][DHCP_DIR_COUNT][DHCP_MESSAGE_TYPE_COUNT])
+static void dhcp_print_counters(const char *vlan_intf,
+                                dhcp_counters_type_t type,
+                                uint64_t counters[][DHCP_MESSAGE_TYPE_COUNT])
 {
-    uint64_t *rx_counters = counters[DHCP_COUNTERS_CURRENT][DHCP_RX];
-    uint64_t *tx_counters = counters[DHCP_COUNTERS_CURRENT][DHCP_TX];
+    static const char *counter_desc[DHCP_COUNTERS_COUNT] = {
+        [DHCP_COUNTERS_CURRENT] = " Current",
+        [DHCP_COUNTERS_SNAPSHOT] = "Snapshot"
+    };
 
     syslog(LOG_NOTICE,
-           "[%*s] DHCP Discover rx/tx: %*lu/%*lu, Offer rx/tx: %*lu/%*lu, "
-           "Request rx/tx: %*lu/%*lu, ACK rx/tx: %*lu/%*lu\n",
+           "[%*s-%*s] Di. rx/tx: %*lu/%*lu, O. rx/tx: %*lu/%*lu, "
+           "Req. rx/tx: %*lu/%*lu, A. rx/tx: %*lu/%*lu\n",
```

Comment:
I'm not a fan of these abbreviations. Makes the log cryptic.

Reply:
OK.